discord/AlignedChatBox.css

```diff
@@ -1,10 +1,10 @@
 /* ----- Aligned Chatbox -----
   Squares up chatbox with user control area */
 :is(main,section)[class^=chatContent] > form,
-form > div[class^=submitContainer] {
+form > div {
```

Comment:
No. You can't just apply styles to every div with a form as it's parent.

Reply:
I made it better